openpype/hosts/max/plugins/create/create_render.py

```diff
@@ -1,8 +1,22 @@
 # -*- coding: utf-8 -*-
 """Creator plugin for creating camera."""
 from openpype.hosts.max.api import plugin
+from openpype.lib import (
+    TextDef,
+    BoolDef,
+    NumberDef,
+)
 from openpype.pipeline import CreatedInstance
 from openpype.hosts.max.api.lib_rendersettings import RenderSettings
+from openpype.settings import get_project_settings
+from openpype.pipeline import legacy_io
+
+
+def setting(project_setting=None):
+    render_setting = get_project_settings(
+        legacy_io.Session["AVALON_PROJECT"]
+    )
+    return render_setting["deadline"]["publish"]["MaxSubmitDeadline"]
```

Comment:
Ouch, do not doing this! :) You're requerying the database each time per attribute definition.

You should instead be setting the defaults using [`apply_settings` on the Creator like here](https://github.com/ynput/OpenPype/blob/173b404d0e2af1712f9e7cf4d9f60ac5ea030570/openpype/hosts/tvpaint/plugins/create/create_review.py#L15-L21)